ple/games/donkeykong/Game.py

```diff
@@ -0,0 +1,197 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import sys
+from pygame.locals import *
+from Board import Board
+from .. import base
+
+'''
+This class defines the logic of the game and how player input is taken etc
+We run one instance of this class at the start of the game, and this instance manages the game for us.
+'''
+
+
+class DonkeyKong(base.Game):
+    def __init__(self):
+        '''
+        Set the height and width for the game element
+        FPS is set to 30 frames per second
+        '''
+        self.height = 520
+        self.width = 1200
+
+        actions = {
+                "left": K_a,
+                "right": K_d,
+                "jump": K_SPACE,
+                "up": K_w,
+                "down": K_s
+        }
+        base.Game.__init__(self, self.width, self.height, actions=actions)
+        self.init()
+
+    def init(self):
+        # Font is set to comic sans MS
+        self.myFont = pygame.font.SysFont("comicsansms", 30)
+
+        # Create a new instance of the Board class
+        self.newGame = Board(self.width, self.height)
```

Comment:
Can you make this accept `self.rewards` as well? I will comment above on the `Board` class on which parts to modify. This lets us adjust the rewards of the game without modifying the game directly.